### DISCUSSION TITLE
Elasticsearch and Kibana are now business risks

### DISCUSSION THREAD

**Discussion by issmonitor** [Security-Onion-Solutions/securityonion#2637] [Unsupported Versions] [answered]
Any thoughts on the change to the Elasticsearch and Kibana SSLP

https://anonymoushash.vmbrasseur.com/2021/01/14/elasticsearch-and-kibana-are-now-business-risks

**Comment by dougburks** [comment, marked as answer]
https://blog.securityonion.net/2021/01/upcoming-elastic-licensing-changes.html
